newsroom/agenda/views.py

```diff
@@ -112,3 +123,13 @@ def follow():
     update_action_list(data.get('items'), 'watches', item_type='agenda')
     push_user_notification('saved_items', count=get_resource_service('agenda').get_saved_items_count())
     return flask.jsonify(), 200
+
+
+@blueprint.route('/agenda_print', methods=['POST'])
+@login_required
+def print_agenda():
```

Comment:
how is this useful?

Reply:
The new shape of the print template has fields like event date&time, contacts, location details and the map which we have the business rules on client side. Instead of creating the same rules for the backend, I wanted to pass that data to backend.

Reply:
I see, imo it would be better then to render the print page in response to that post and avoid the cache